### ISSUE TITLE
Unable to obtain repeatable results 

### ISSUE THREAD

**Issue by Jiaxuan-Peng** [yandex-research/rtdl-revisiting-models#45]
Hi,

Thank you for your nice work! I'm writing to make sure how to generate repeatable results. 

Followed by https://github.com/yandex-research/rtdl-revisiting-models/blob/main/package/example.ipynb, I used FFtransformer based on my dataset, which is a multi class task and I changed the loss function as Cross-Entropy. 

I have set the seed as the ipynb did, that is, delu.random.seed(0)  Also, I set random.seed(0), np.random(0), torch.manual_seed(0). But, the results still changed every time. Do you have any clue about this？

Or, maybe I should save the trained model to avoid this issue？If you could show the code about how to save the model, that would definitely be helpful. Thanks. 

Best,
Jiaxuan

**Comment by Yura52** [comment]
Hi @Jiaxuan-Peng ! Have you resolved the issue? Just curious what turned out to be the root cause of the problem and how you fixed it.

**Comment by Jiaxuan-Peng** [comment]
Hi, I resolved it! It's because I put the code of this model into two parts in the ipynb file and when I fine-tuning it, I only run the second part. The root cause may be that the seed is located in the first part.

The first part ends with the following code.
`# >>> Labels.
# Regression labels must be represented by float32.
if task_type == "regression":
    Y = Y.astype(np.float32)
else:
    assert n_classes is not None
    Y = Y.astype(np.int64)
    assert set(Y.tolist()) == set(
        range(n_classes)
    ), "Classification labels must form the range [0, 1, ..., n_classes - 1]"
`

The second part starts with `d_out = n_classes if task_type == "multiclass" else 1
model = FTTransformer(
    n_cont_features=n_cont_features,
    cat_cardinalities=cat_cardinalities,
    d_out=d_out,
    **FTTransformer.get_default_kwargs(),
).to(device)
optimizer = model.make_default_optimizer()`
